.goreleaser.yml

```diff
@@ -0,0 +1,81 @@
+before:
+  hooks: ["go mod tidy"]
+
+build_common: &build_common
+  main: "./cmd/zed"
+  binary: "zed"
+  env:
+    - "CGO_ENABLED=1"
+  mod_timestamp: "{{ .CommitTimestamp }}"
+  asmflags: ["all=-trimpath={{ .Env.PWD }}"]
+  gcflags: ["all=-trimpath={{ .Env.PWD }}"]
+  ldflags:
+    - "-s -w"
+    - "-extldflags=-static"
+    - "-X {{ .Env.PKG }}/version.Version={{ .Version }}"
+
+builds:
+  - id: "linux-amd64"
+    <<: *build_common
+    goos: ["linux"]
+    goarch: ["amd64"]
+    env: ["CC=gcc"]
+  - id: "linux-arm64"
+    <<: *build_common
+    goos: ["linux"]
+    goarch: ["arm64"]
+    env: ["CC=aarch64-linux-gnu-gcc"]
+  - id: "windows-amd64"
+    <<: *build_common
+    goos: ["windows"]
+    goarch: ["amd64"]
+    env: ["CC=x86_64-w64-mingw32-gcc-posix"]
+
+nfpms:
+  - vendor: "authzed inc."
+    homepage: "https://authzed.com/"
+    maintainer: "authzed <support@authzed.com>"
+    description: "manage Authzed from your command line."
+    license: "Apache 2.0"
+    epoch: 0
+    formats:
+      - "apk"
+      - "deb"
+      - "rpm"
+
+dockers:
+  - image_templates:
+      - "quay.io/authzed/zed:{{ .Version }}-amd64"
+    ids: ["linux-amd64"]
+    goos: "linux"
+    goarch: "amd64"
+    use: "buildx"
+    build_flag_templates:
+      - "--platform=linux/amd64"
+  - image_templates:
+      - "quay.io/authzed/zed:{{ .Version }}-arm64"
+    ids: ["linux-arm64"]
+    goos: "linux"
+    goarch: "arm64"
+    use: "buildx"
+    build_flag_templates:
+      - "--platform=linux/arm64"
+
+docker_manifests:
```

Comment:
does this produce a manifest list?

Reply:
yep, out of the images built in the docker step